backend/ttrs/serializers.py

```diff
@@ -59,3 +59,46 @@ def validate(self, data):
             raise ValidationError({'password': ve.messages})
         data['password'] = make_password(data['password'])
         return data
+
+
+class CourseSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Course
+        fields = '__all__'
+
+
+class LectureSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Lecture
+        fields = '__all__'
+
+
+class CollegeSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = College
+        fields = '__all__'
+
+
+class DepartmentSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Department
+        fields = '__all__'
+
+
+class MajorSerializer(serializers.ModelSerializer):
+    college = serializers.SerializerMethodField()
```

Comment:
The model `Major` doesn't have `college` field. So, I added `SerializerMethodField` so that we can figure out which college the major is belong to.